DigitalLearningSolutions.Web/ViewModels/TrackingSystem/Centre/Configuration/RegistrationPrompts/AddRegistrationPromptConfigureAnswersViewModel.cs

```diff
@@ -24,5 +28,51 @@ public class RegistrationPromptAnswersViewModel
         public string? Answer { get; set; }
 
         public bool IncludeAnswersTableCaption { get; set; }
+
+        private IEnumerable<string> ComparableOptions => NewlineSeparatedStringListHelper
+            .SplitNewlineSeparatedList(OptionsString)
+            .Select(o => o.Trim().ToLower());
+
+        public IEnumerable<ValidationResult> Validate(ValidationContext validationContext)
+        {
+            var validationResults = new List<ValidationResult>();
+
+            if (AnswerInOptionsString())
+            {
+                validationResults.Add(
+                    new ValidationResult(
+                        "That answer is already in the list of options.",
+                        new[]
+                        {
+                            nameof(Answer),
+                        }
+                    )
+                );
+            }
+
+            if (OptionsStringContainsDuplicates())
+            {
+                validationResults.Add(
+                    new ValidationResult(
+                        "The list of answers contains duplicate options.",
+                        new string[] { }
+                    )
+                );
+            }

```

Comment:
Gave it a quick test, and this validation appears to work in the Edit journey but not the Add journey. I think you need to check if the model state is invalid in the AddRegistrationPromptConfigureAnswersPostNext method in the controller?